### ISSUE TITLE
Make sure number of closed issues it displayed

### ISSUE THREAD

**Issue by konstantinast** [konstantinast/fullstack-party#5]
At the moment button does not show Closed Issue amount as Github v3 API does not have the ability to closed_issue_count for /owner/repo/issues like the website search can and for that matter API /search method also has no way to satisfy ``owner && repo``  condition (or at least I was not able to find it).

Buuuuuut after Googling/Githubing (is that even a word) for a while was able to find this issue https://github.com/isaacs/github/issues/536 , where they recommend to use Github v4 (GraphQL) API. Everything would be fine and dandy, but Github only allows to use v4 API with authorization token. 

In other words I'm blocked by this issue https://github.com/konstantinast/fullstack-party/issues/4

**Comment by konstantinast** [comment]
done
